s3-source.tf

```diff
@@ -49,6 +49,47 @@ data "aws_iam_policy_document" "source_replication_policy" {
       "${local.dest_bucket_object_arn}",
     ]
   }
+
+  statement {
+    action = [
+      "s3:InitiateReplication"
+    ]
+    resource = [
+      "${local.dest_bucket_object_arn}",
+    ]
+  }
+
+  statement {
+    action = [
+      "s3:GetReplicationConfiguration",
+      "s3:PutInventoryConfiguration"
+    ]
+
+    resource = [
+      "${local.dest_bucket_object_arn}",
+    ]
+  }
+
+  statement {
+    action = [
+      "s3:GetObject",
+      "s3:GetObjectVersion"
+    ]
+
+    resource = [
+      "${local.dest_bucket_object_arn}",
+    ]
+  }
+
+  statement {
+    action = [
+      "s3:PutObject"
+    ]
+
+    resource = [
+      "${local.dest_bucket_object_arn}",
```

Comment:
I think the manifest bucket is the source bucket so can probably throw in with `${local.source_bucket_object_arn}` stuff. (notice the `/*` at the end indicating it's for objects in the bucket).